logstash-input-kafka.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-input-kafka'
-  s.version         = '2.0.2'
+  s.version         = '2.1.0'
```

Comment:
Oh, this needs to be 3.0.0. It breaks compatibility. We are removing 90% of the old configuration :)